service/src/main/kotlin/fi/espoo/evaka/reports/AssistanceNeedDecisionsReport.kt

```diff
@@ -26,11 +28,12 @@ class AssistanceNeedDecisionsReport(private val accessControl: AccessControl, pr
     ): List<AssistanceNeedDecisionsReportRow> {
         Audit.AssistanceNeedDecisionsReportRead.log()
         accessControl.requirePermissionFor(user, Action.Global.READ_ASSISTANCE_NEED_DECISIONS_REPORT)
+        val authorizedUnits = acl.getAuthorizedUnits(user, setOf(UserRole.UNIT_SUPERVISOR))
```

Comment:
`setOf(UserRole.UNIT_SUPERVISOR)` probably should be customizable but where? Is it possible to read it from `accessControl`?

Reply:
Rebase, and use the authorization filter technique as here: https://github.com/espoon-voltti/evaka/blob/ddfef2fd7fa5376fb77e71c0f9116bfa1932e4b6/service/src/main/kotlin/fi/espoo/evaka/daycare/controllers/DaycareController.kt#L64

You'll need to add and use a unit-level action (`Action.Unit.READ_ASSISTANCE_NEED_DECISIONS_REPORT`). Unfortunately we can't get rid of the global action yet, because the reports page uses it for visibility. There's no need to do actual access control with the global action anymore, but it should have "compatible" rules so that the report appears correctly on the page.

Reply:
Done!